enumerable.rb

```diff
@@ -0,0 +1,110 @@
+# rubocop:disable Metrics/CyclomaticComplexity
+# rubocop:disable Metrics/PerceivedComplexity
+
+module Enumerable
+  def my_each
+    return to_enum unless block_given
+
+    arr.length.times do |n|
+      yield arr[n]
+    end
+  end
```

Comment:
- [ ] enumerable #my_each when a block is given when self is a range returns the range itself after calling the given block once for each element in self

Here is the error I am getting
<img width="666" alt="1" src="https://user-images.githubusercontent.com/58504655/103562602-2d51b300-4edd-11eb-9f2b-3cfe8d7e7ae5.PNG">